common/app/views/support/Profile.scala

```diff
@@ -200,7 +200,7 @@ object ImgSrc extends Logging with implicits.Strings {
     }
   }
 
-  private def findLargestSrc(ImageElement: ImageMedia, profile: Profile): Option[String] = {
+  def findLargestSrc(ImageElement: ImageMedia, profile: Profile): Option[String] = {
```

Comment:
Why was it private before? Would the proper function to use be `srcsetForProfile`?